Library/Homebrew/tap_migrations.rb

```diff
@@ -38,6 +40,7 @@
   "electric-fence" => "homebrew/boneyard",
   "fceux" => "homebrew/games",
   "feh" => "homebrew/x11",
+  "figtoipe" => "homebrew/head-only",
```

Comment:
This seems to only live now in the "ipe-tools" GitHub repo. (See pull in head-only.)